ResourceStatusSystem/Agent/SiteInspectorAgent.py

```diff
@@ -177,11 +177,11 @@ def _execute(self):
       except Queue.Empty:
         return S_OK()
 
-      resEnforce = pep.enforce(site)
-      if not resEnforce['OK']:
-        self.log.error('Failed policy enforcement', resEnforce['Message'])
-        self.sitesToBeChecked.task_done()
```

Comment:
Are you missing this line?

Reply:
No, it is there in line 187. 

As there was no code between `continue` and `task_done` in line 187, I dropped `continue` and this `task_done` here.

One could argue that it would be more defensive and future proof to keep this line and the `continue`, in case someone starts putting some code between the error of enforcement and the end of the loop.